Ideas-2020.md

```diff
@@ -184,12 +175,9 @@ Proposal](https://github.com/sugarlabs/musicblocks/issues/2058).
 **Suggested issues to work on:**<br>
  - [https://github.com/sugarlabs/musicblocks/issues/2020](https://github.com/sugarlabs/musicblocks/issues/2020)
 
-**Coding Mentors**<br>
+**Mentors**<br>
 [Walter Bender](https://github.com/walterbender) and [Sumit Srivastava](https://github.com/sum2it).
 
```

Comment:
I do not know how this could possibly be completed without my assistance.

Reply:
Yes.  You must stay involved for this project to be completed.

Reply:
I'd recommend having Walter and Devin as mentors for this project.